frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/storage-tab-state-update.ts

```diff
@@ -12,10 +12,23 @@ import { StorageUISource } from '../../../../modals/disk-modal/storage-ui-source
 import { getNextIDResolver } from '../../../../../utils/utils';
 import { getStorages } from '../../../selectors/selectors';
 import { vmWizardInternalActions } from '../../internal-actions';
+import { getTemplateValidations } from '../../../selectors/template';
 
 export const prefillInitialDiskUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   const state = getState();
-  if (!hasVmSettingsChanged(prevState, state, id, VMSettingsField.PROVISION_SOURCE_TYPE)) {
+
+  const prevValidations = getTemplateValidations(prevState, id);
+  const tValidations = getTemplateValidations(state, id);
```

Comment:
there is a `getTemplateValidation` function

Reply:
can you rename the variable to `nextValidations`?

Reply:
I guess we should use the same terminology everywhere - please see oldType vs newType. Can you unite these somehow?

Reply:
see "oldType vs newType" where ?

Reply:
here inside prefillInitialDiskUpdater